src/DataProtection/DataProtection/src/Managed/AesGcmAuthenticatedEncryptor.cs

```diff
@@ -13,7 +13,7 @@
 namespace Microsoft.AspNetCore.DataProtection.Managed
 {
     // An encryptor that uses AesGcm to do encryption
-    internal unsafe sealed class AesGcmAuthenticatedEncryptor : IOptimizedAuthenticatedEncryptor, IDisposable
+    internal unsafe sealed class AesGcmAuthenticatedEncryptor : ISpanAuthenticatedEncryptor, IDisposable
```

Comment:
This was the new managed implementation we added in preview2 in the last PR, so it seems reasonable that it can use span only codepaths since its NetCoreApp only already, and as a result can drop the other internal optimized interface that it was implementing